plugins/network-elements/nuage-vsp/src/net/nuage/vsp/acs/client/NuageVspApiClient.java

```diff
@@ -23,8 +23,8 @@
 
     public void login() throws Exception;
 
-    public void setNuageVspHost(String restRelativePath, String[] cmsUserInfo, int noofRetry, int retryInterval);
+    public void setNuageVspHost(String restRelativePath, String[] cmsUserInfo, int noofRetry, int retryInterval, String nuageVspCmsId);
 
     public String executeRestApi(String method, String resource, String resourceId, String childResource, Object entityDetails, String resourceFilter, String proxyUserUuid,
-            String proxyUserDomainuuid) throws Exception;
+                                 String proxyUserDomainuuid) throws Exception;
```

Comment:
The private client is using custom exceptions which are translated to global exceptions for use inside the CS plugin